cyral/resource_cyral_repository.go

```diff
@@ -16,8 +16,41 @@ import (
 const (
 	mongodbRepoType             = "mongodb"
 	mongodbReplicaSetServerType = "replicaset"
+	repositoryTypesMarkdown     = "\n  - `bigquery`" +
+		"\n  - `cassandra`" +
+		"\n  - `denodo`" +
+		"\n  - `dremio`" +
+		"\n  - `galera`" +
+		"\n  - `mariadb`" +
+		"\n  - `mongodb`" +
+		"\n  - `mysql`" +
+		"\n  - `oracle`" +
+		"\n  - `postgresql`" +
+		"\n  - `redshift`" +
+		"\n  - `s3`" +
+		"\n  - `snowflake`" +
+		"\n  - `sqlserver`"
 )
 
+func repositoryTypes() []string {
+	return []string{
+		"bigquery",
+		"cassandra",
+		"denodo",
+		"dremio",
+		"galera",
+		"mariadb",
+		"mongodb",
+		"mysql",
+		"oracle",
+		"postgresql",
+		"redshift",
+		"snowflake",
+		"s3",
```

Comment:
```suggestion
		"s3",
		"snowflake",
```

Put `s3` before `snowflake` so we can have the same order in `repositoryTypesMarkdown`.